packages/opencrvs/src/Adaptor.js

```diff
@@ -0,0 +1,202 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+import { execute as commonExecute } from '@openfn/language-common';
+import { searchEventsQuery } from './query';
+
+/**
+ * State object
+ * @typedef {Object} OpenCRVSState
+ * @property data - the parsed response body
+ * @property response - the response from the HTTP server, including headers, statusCode, body, etc
+ * @property references - an array of all previous data objects used in the Job
+ **/
+
+/**
+ * Execute a sequence of operations.
+ * Wraps `language-common/execute` to make working with this API easier.
+ * @example
+ * execute(
+ *   create('foo'),
+ *   delete('bar')
+ * )(state)
+ * @private
+ * @param {Operations} operations - Operations to be performed.
+ * @returns {Operation}
+ */
+export function execute(...operations) {
+  const initialState = {
+    references: [],
+    data: null,
+  };
+
+  return state => {
+    return commonExecute(
+      util.authorize,
+      ...operations
+    )({
+      ...initialState,
+      ...state,
+    });
+  };
+}
+
+/**
+ * Create a Birth Notification
+ * @example
+ * createBirthNotification([
+  {
+    fullUrl: 'urn:uuid:eee21c26-67a2-40af-8cf7-5f4bc969153f',
+    resource: {
+      resourceType: 'QuestionnaireResponse',
+      extension: [],
+      status: 'completed',
+      subject: {
+        reference: 'urn:uuid:7cb1d9cc-ea4b-4046-bea0-38bdf3082f56',
+      },
+      item: [
+        {
+          text: 'birth.mother.mother-view-group.motherIdType',
+          linkId: '',
+          answer: [
+            {
+              valueString: 'NATIONAL_ID',
+            },
+          ],
+        },
+        {
+          text: 'birth.father.father-view-group.fatherIdType',
+          linkId: '',
+          answer: [
+            {
+              valueString: 'NATIONAL_ID',
+            },
+          ],
+        },
+      ],
+    },
+  },
+]);
+
+ * @function
+ * @public
+ * @param {Array} body - An array of Birth Notification FHIR resources
```

Comment:
Can we list (here or in the main docs body) the required resources? And maybe link to the postman JSON?

This isn't super great practice for documentation, but I think it's OK in this instance